### DISCUSSION TITLE
Missing From Equipment Compendium

### DISCUSSION THREAD

**Discussion by CptTwinkie** [wildj79/foundryvtt-starfinder#349] [General]
Grenade arrow I
Grenade arrow II
Grenade arrow III
Grenade arrow IV
Tactical missile
Advanced missile
Lock, Simple
Lock, Average
Lock, Good
Lock, Superior
Medical lab
Portable Light, Flashlight
Portable Light, Lantern
Portable Light, Beacon
Portable Light, Spotlight
Binders
Manacles
Signal Jammer
Grappler
Holoskin
Laser microphone
Spy drone
Motion detector
X-ray visor
Regeneration table
Pearly white spindle
Spell Ampoule 0
Spell Ampoule 1st
Spell Ampoule 2nd
Spell Ampoule 3rd
Spell Gem 0
Spell Gem 1st
Spell Gem 2nd
Spell Gem 3rd
Spell Gem 4th
Spell Gem 5th
Spell Gem 6th
Tiara of Translocation Mk 1
Tiara of Translocation Mk 2
Tiara of Translocation Mk 3
Tiara of Translocation Mk 4
Ring of sustenance
Staff of mystic healing
Glove of storing
Rod of cancellation
Shadow orb
Ring of cosmic alignment
Serum of sex shift
Serum of enhancement, Commando
Serum of enhancement, Diplomat
Serum of enhancement, Scientist
Serum of enhancement, Sensate
Serum of enhancement, Sneak
Serum of enhancement, Warrior
Starstone compass
Mindlink circlet, Mk 1
Mindlink circlet, Mk 2
Mindlink circlet, Mk 3
Efficient bandolier
Psychic booster
Digital harrow deck
Mnemonic editor, mk 1
Mnemonic editor, mk 2
Mnemonic editor, mk 3
Mnemonic editor, mk 4
Clothing, Environmental
Clothing, Everyday
Clothing, Formal
Clothing, Professional
Clothing, Travel
Tent, Mass produced
Tent, Mobile hotelier
Bonding epoxy
Gear clamp
Hygiene kit
Hyperleaf
Dreamshiver
Transdimensional pesh
Megaopiate
Medicinals, Tier 1
Medicinals, Tier 2
Medicinals, Tier 3
Medicinals, Tier 4
Id moss
Insanity mist
Blue whinnis
Green lotus extract
Ungol dust
Black lotus extract
Deathblade
Fine art
Gem
Grain
Textiles, common
Textiles, fine
UPB (1000)
Meal, Poor
Meal, Common
Meal, Good

**Comment by CptTwinkie** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Electrostatic Fields Missing From Equipment Compendium](https://trello.com/c/edLiq41i/64-electrostatic-fields-missing-from-equipment-compendium)

**Comment by TimToxopeus** [comment]
Some of these items have been added to the compendium, can you verify which ones are still missing and should be added?

**Comment by CptTwinkie** [comment]
list updated
